pkg/trace/schema/mempool.go

```diff
@@ -2,7 +2,6 @@ package schema
 
 import (
 	"github.com/cometbft/cometbft/libs/bytes"
-	"github.com/cometbft/cometbft/p2p"
```

Comment:
got rid of most imports to the schema package so we can import in the p2p packages